notification-channel-config-service-impl/src/main/java/org/hypertrace/notification/config/service/NotificationChannelConfigServiceRequestValidator.java

```diff
@@ -20,17 +22,50 @@
 
 public class NotificationChannelConfigServiceRequestValidator {
 
+  public static final String WEBHOOK_EXCLUSION_DOMAINS = "webhook.exclusion.domains";
+
   public void validateCreateNotificationChannelRequest(
-      RequestContext requestContext, CreateNotificationChannelRequest request) {
+      RequestContext requestContext,
+      CreateNotificationChannelRequest request,
+      Config notificationChannelConfig) {
     validateRequestContextOrThrow(requestContext);
     validateNotificationChannelMutableData(request.getNotificationChannelMutableData());
+    validateWebhookConfigExclusionDomains(
```

Comment:
Please add unit tests

Reply:
done. Added validation for https support